src/config/strings.js

```diff
@@ -65,6 +65,8 @@ export default {
       insufficientBalanceErrorMsg: 'Insufficient balance',
       maxDepositErrorMsg:
         'You have exceeded the maximum transfer amount ({{maxDeposit}} {{symbol}}). Please reduce the amount and try again.',
+      bridgeIsFullErrorMsg: 'The token bridge is currently full, try another token',
+      bridgeIsFullReadMore: 'read more',
```

Comment:
I would start to move the generic texts to the component level (as a preparation to the commons repo)